src/main/java/org/kohsuke/github/GHPullRequestReviewState.java

```diff
@@ -0,0 +1,19 @@
+package org.kohsuke.github;
+
+public enum GHPullRequestReviewState {
+    PENDING(null),
+    APPROVED("APPROVE"),
```

Comment:
Isn't this a typo `APPROVE` -> `APPROVED`?  At least GH Api docs are listing the `APPROVED` state https://developer.github.com/v3/pulls/reviews/#get-a-single-review